dev-infra/release/publish/actions/move-next-into-feature-freeze.ts

```diff
@@ -6,103 +6,21 @@
-    await this.verifyPassingGithubStatus(nextBranch);
-    await this.checkoutUpstreamBranch(nextBranch);
-    await this.createLocalBranchFromHead(newBranch);
-    await this.pushHeadToRemoteBranch(newBranch);
-    info(green(`  ✓   Version branch "${newBranch}" created.`));
-  }
-
-  /**
-   * Creates a pull request for the next branch that bumps the version to the next
-   * minor, and cherry-picks the changelog for the newly branched-off feature-freeze version.
-   */
-  private async _createNextBranchUpdatePullRequest(
-      releaseNotes: ReleaseNotes, newVersion: semver.SemVer) {
-    const {branchName: nextBranch, version} = this.active.next;
-    // We increase the version for the next branch to the next minor. The team can decide
-    // later if they want next to be a major through the `Configure Next as Major` release action.
-    const newNextVersion = semver.parse(`${version.major}.${version.minor + 1}.0-next.0`)!;
-    const bumpCommitMessage = getCommitMessageForExceptionalNextVersionBump(newNextVersion);
-
-    await this.checkoutUpstreamBranch(nextBranch);
-    await this.updateProjectVersion(newNextVersion);
-
-    // Create an individual commit for the next version bump. The changelog should go into
-    // a separate commit that makes it clear where the changelog is cherry-picked from.
-    await this.createCommit(bumpCommitMessage, [packageJsonPath]);
-
-    await this.prependReleaseNotesToChangelog(releaseNotes);
-
-    const commitMessage = getReleaseNoteCherryPickCommitMessage(releaseNotes.version);
-
-    await this.createCommit(commitMessage, [changelogPath]);
-
-    let nextPullRequestMessage = `The previous "next" release-train has moved into the ` +
-        `release-candidate phase. This PR updates the next branch to the subsequent ` +
-        `release-train.\n\nAlso this PR cherry-picks the changelog for ` +
-        `v${newVersion} into the ${nextBranch} branch so that the changelog is up to date.`;
-
-    const nextUpdatePullRequest = await this.pushChangesToForkAndCreatePullRequest(
-        nextBranch, `next-release-train-${newNextVersion}`,
-        `Update next branch to reflect new release-train "v${newNextVersion}".`,
-        nextPullRequestMessage);
-
-    info(green(`  ✓   Pull request for updating the "${nextBranch}" branch has been created.`));
-    info(yellow(`      Please ask team members to review: ${nextUpdatePullRequest.url}.`));
-  }
+export class MoveNextIntoFeatureFreezeAction extends BranchOffNextBranchBaseAction {
+  override newPhaseName = 'feature-freeze' as const;
 
   static override async isActive(active: ActiveReleaseTrains) {
-    // A new feature-freeze/release-candidate branch can only be created if there
-    // is no active release-train in feature-freeze/release-candidate phase.
+    // A new feature-freeze branch can only be created if there is no active
+    // release-train in feature-freeze/release-candidate phase.
     return active.releaseCandidate === null;
```

Comment:
We should have this only be active when its possible to move into feature freeze, which means only cases where next is major.

```suggestion
    return active.releaseCandidate === null && active.next.isMajor;
```

Reply:
I was thinking it's just an option for a minor to move directly into RC, but it should be still possible to go into feature freeze.

or do you think we should not permit feature freeze for minors at all?

Reply:
I think we shouldn't permit it at all as we don't plan to use feature-freezes in minors in the future either

Reply:
Done. Makes sense! we can always adjust if needed.